docs/howto/composite-primary-key.txt

```diff
@@ -66,7 +66,21 @@ How to define a composite foreign key
 At the moment, relationship fields (``ForeignKey``, ``OneToOneField``,
 ``ManyToManyField``) don't support composite primary keys.
 
-For the time being, the ``ForeignObject`` field can be used as an alternative:
+So, given the ``Tenant`` and ``User`` models, we cannot do:
+
+.. code-block:: pycon
+
+    class Comment(models.Model):
+        pk = models.CompositePrimaryKey("tenant_id", "id")
+        tenant = models.ForeignKey(Tenant, on_delete=models.CASCADE)
+        id = models.IntegerField()
```

Comment:
I think giving `Comment` a composite primary key is a distraction from the point about `ForeignKey`.

Reply:
Yeah, fair point I adjusted it